repo.go

```diff
@@ -338,3 +282,35 @@ func (f *File) Contents() string {
 	}
 	return content
 }
+
+func (f *File) getBlame() (*git.BlameResult, error) {
+	blame, err := git.Blame(f.headCommit, f.file.Name)
```

Comment:
With the current implementation of `GetAllFiles` this will nil pointer dereference and panic, should probably check headCommit isn't nil right?

Reply:
This shouldn't be the case anymore, do you still think it's worth adding the check?

Reply:
Now that tests cover this I think we should be fine without